matrix-neoboard-widget/src/AppContainer.tsx

```diff
@@ -57,9 +57,9 @@ export const AppContainer = ({
             <MuiWidgetApiProvider
               widgetApiPromise={widgetApiPromise}
               widgetRegistration={{
-                name: 'NeoBoard',
-                // "pad" suffix to get a custom icon
-                type: 'net.nordeck.whiteboard:pad',
+                name: 'NeoMap',
+                // "map" suffix to get a custom icon
+                type: 'net.nordeck.neomap:map',
```

Comment:
Is this intentional? 🤔

Reply:
No

Reply:
Fixed in 176ef0729f9bc77d69f06606ba6847fe45648a83